en/components/input-group.md

```diff
@@ -131,6 +131,142 @@ The Ignite UI for Angular Input Group component can be used inside strictly type
+</code-view>
+
+### Reactive Forms
+Reactive form validation is achieved by adding validator functions directly to the form control model in the component class. After creating the control in the component class, it should be associated with a form control element in the template.
+
+```ts
+public registrationForm = new FormGroup<User>({
+    username: new FormControl<string>('', { nonNullable: true, validators: [Validators.required] }),
+    email: new FormControl<string>('', { nonNullable: true, validators: [Validators.required, Validators.email] }),
+    password: new FormControl<string>('', { nonNullable: true, validators: [Validators.required, Validators.minLength(8)] })
+});
+```
+
+```html
+<form [formGroup]="registrationForm">
+    <igx-input-group>
+        <label igxLabel for="username">Username</label>
+        <input igxInput name="username" type="text" formControlName="username" />
+    </igx-input-group>
+
+    <igx-input-group>
+        <label igxLabel for="email">Email</label>
+        <input igxInput name="email" type="email" formControlName="email" />
+    </igx-input-group>
+
+    <igx-input-group>
+        <label igxLabel for="password">Password</label>
+        <input igxInput name="password" type="password" formControlName="password" />
+    </igx-input-group>
+</form>
+```
+
+Similar to the template-driven form sample, when having additional validation like `email` and `minLength`, an [`igx-hint`]({environment:angularApiUrl}/classes/igxhintdirective.html) directive could be used to notify the user if the validaton fails.
+
+The following example demonstrates how to access the control through a `get` method and inspect its state.
+
+```ts
+get email() {
+    return this.registrationForm.get('email');
+}
+get password() {
+    return this.registrationForm.get('password');
+}
+```
+
+```html
+<igx-input-group>
+    <label igxLabel for="email">Email</label>
+    <input igxInput name="email" type="email" formControlName="email" />
+    <igx-hint *ngIf="email.errors?.email">Please enter a valid email</igx-hint>
+</igx-input-group>
+
+<igx-input-group>
+    <label igxLabel for="password">Password</label>
+    <input igxInput name="password" type="password" formControlName="password" />
+    <igx-hint *ngIf="password.errors?.minlength">Password should be at least 8 characters</igx-hint>
+</igx-input-group>
+```
+
+The result from the above configurations could be seen in the below sample.
+
+<code-view style="height:480px"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/data-entries/reactive-form-validation" >
+</code-view>
+
+### Custom Validators
+Some input fields may require custom validation and this could be achieved via custom validators. When the value is valid, the validator will return `null` and when the value is invalid, it will generate a set of errors that could be used to display a specific error message on the screen.
```

Comment:
I have left a review comment about the custom validation sample in the samples repo. There, you would be able to create an example of how one from control drives the validity of a different form control, or of the entire group.